### PR TITLE
Rad-2733 Fix usage of AuthenticationPrincipal

### PR DESCRIPTION
**Pull request by MertCingoz**

### CODE REVIEW

**Review by benji811124 — APPROVED** [review]
Looks good to me

File: Mango API/src/com/infiniteautomation/mango/rest/latest/UserCommentRestController.java [hunk 1]
```diff
@@ -113,8 +114,10 @@ public ResponseEntity<UserCommentModel> createNewUserComment(
 
         //Assign a userId if there isn't one
         if(model.getUserId() <= 0){
```
Comment (jazdw) [inline Mango API/src/com/infiniteautomation/mango/rest/latest/UserCommentRestController.java:116]: This seems odd to me, wouldn't this allow anyone to post a comment as any other user?

**Comment by jazdw** [comment]
Looks good otherwise @MertCingoz

File: Mango API/src/com/infiniteautomation/mango/rest/latest/UserCommentRestController.java (hunk 1 above)
Comment (MertCingoz) [reply Mango API/src/com/infiniteautomation/mango/rest/latest/UserCommentRestController.java:116]: There is also delete/update/query methods which is not used by UI.
I think initial requirement of this feature and current usage is completely different.

**Pull request merged**